src/main/java/io/openliberty/tools/common/plugins/util/AbstractContainerSupportUtil.java

```diff
@@ -21,21 +45,81 @@ public abstract class AbstractContainerSupportUtil {
      */
     public abstract void error(String msg, Throwable e);
 
+    /**
+     * Log info
+     * @param msg
+     */
+    public abstract void info(String msg);
+
+    protected String getContainerCommandPrefix() throws PluginExecutionException {
+        if (!checkedContainerType) {
+            checkDockerVersion();
+        }
+
+        if (!isDocker) {
+            return CONTAINER_PODMAN_PREFIX;
+        }
+
+        return CONTAINER_DOCKER_PREFIX;
+    }
+
+    /**
+     * Retrieve the current docker version and compare to a known value.
+     * The Maven class ComparableVersion allows for numbers, letters and certain words.
+     * Throw an exception if there is a problem with the version.
+     */
+    private static final String MIN_DOCKER_VERSION = "18.03.0"; // Must use Docker 18.03.0 or higher
+    protected void checkDockerVersion() throws PluginExecutionException {
+        String versionCmd = "docker version --format {{.Client.Version}}";
+        String dockerVersion = execContainerCmd(versionCmd, DOCKER_TIMEOUT);
+        if (dockerVersion == null) {
+            checkPodmanVersion(); // Check Podman version if no Docker 
+            return;
+        }
+        debug("Detected Docker version >" + dockerVersion);
+        
+        if (VersionUtility.compareArtifactVersion(dockerVersion, MIN_DOCKER_VERSION, false) < 0) {
+            checkPodmanVersion(); // Check that bad Docker version isn't just a Podman version
+            if (!isDocker) {
+                return;
+            }
+            throw new PluginExecutionException("The detected Docker client version number is not supported:" + dockerVersion.trim() + ". Docker version must be 18.03.0 or higher.");
```

Comment:
Use the constant in the exception for the minimum version in case it changes in the future. Then we only have to change in one place.